debug_scripts/create-new-eeprom-with-self-signed-certs.sh

```diff
@@ -33,6 +33,7 @@ getEdgeStatus() {
 readEeprom() {
   if [ -f ${IDENTITY_DIR}/identity.json ] ; then
     deviceID=$(jq -r .deviceID ${IDENTITY_DIR}/identity.json)
+    serialNumber=$(jq -r .serialNumber ${IDENTITY_DIR}/identity.json)
```

Comment:
:+1: 

If you end up needing to read one or two more variables from that file, I can show a neat trick that can set multiple variables at once.

Reply:
Sure. Actually this is a temp fix, we are moving away from jq dependency.